package/gluon-ddhcpd/files/lib/gluon/upgrade/317-ddhcp-init-remove

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+rm /etc/init.d/ddhcpd
```

Comment:
maybe check for existence before removal? or does it always exist on upgrades?

Reply:
the ddhcpd package brings an init script nowadays. I am not sure how to best deal with it. removing it does not feel right. However, it is in the way if we don't remove it.

Reply:
Either use `rm -f` or `[[ -f /etc/init.d/ddhcpd ]] && rm /etc/init.d/ddhcpd`. I'd prefer variant 1.

Reply:
The correct way to deal with this is to have an 'enabled' setting in a config file of ddhcpd and disable that. If it doesn't, this needs to be fixed in the ddhcpd package first.

Reply:
done.